src/averageAges.js

```diff
@@ -14,8 +14,20 @@
  *
  * @return {number}
  */
+function calculatePeopleAverageAge(people) {
+  const ages = people.map(person => person.died - person.born);
+
+  return ages.reduce((sum, age) => sum + age, 0) / ages.length;
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const onlyMen = century
+    ? people.filter(men => men.sex === 'm'
+      && Math.ceil(men.died / 100) === century)
+    : people.filter(men => men.sex === 'm');
```

Comment:
No need to duplicate filter methods, just choose needed condition to return inside it